lighthouse-core/scripts/gcp-collection/gcp-setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
since most of this script is `sudo`, is it possible to run the script as sudo from the gcloud invocation? or `su -`?

Reply:
It is, but I'm not sure I'd want to :)

I personally find the explicitness of what requires root permissions and making that set as small as what's required to be a good thing.

Is the repetition your primary concern?